fonts.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+	font-family: 'GothaPro';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GerKateAxion/pullRequest?prid=5991795)